analytic_engine/src/lib.rs

```diff
@@ -120,6 +125,28 @@ pub struct Config {
     pub metrics: MetricsOptions,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(default)]
+pub struct ParquetReaderOptions {
+    enable_page_filter: bool,
+    enable_lazy_row_filter: bool,
+}
+
+impl Default for ParquetReaderOptions {
+    fn default() -> Self {
+        Self {
+            enable_page_filter: true,
+            enable_lazy_row_filter: true,
+        }
+    }
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(tag = "type")]
+pub enum ReaderOptions {
```

Comment:
I guess it should not use enumeration here, because in the future multiple readers may be used all. And I suggest it to be:
```rust
pub struct SstReaderOptions {
    pub enable_parquet_page_filter: bool,
    pub enable_parquet_row_filter: bool,
}
```